testing/cli/test_cli_sct_merge_images.py

```diff
@@ -13,4 +13,4 @@ def test_sct_merge_images_no_checks():
     TODO: Check the results. (This test replaces the 'sct_testing' test, which did not implement any checks.)"""
     sct_merge_images.main(argv=['-i', 'template/template/PAM50_small_cord.nii.gz', 't2/t2_seg-manual.nii.gz',
                                 '-w', 'mt/warp_template2mt.nii.gz', 't2/warp_template2anat.nii.gz',
-                                '-d' 'mt/mt1.nii.gz'])
+                                '-d', 'mt/mt1.nii.gz'])
```

Comment:
I was curious how the tests were even passing if they were broken... :thinking: 

Without the comma, the two strings get concatenated together like this:

![image](https://user-images.githubusercontent.com/16181459/200356905-e2b9ca65-2c32-4a01-a925-42cb49ab756e.png)

But it looks like argparse is smart enough to parse `-dmt/mt1.nii.gz` as `-d mt/mt1.nii.gz`:

![image](https://user-images.githubusercontent.com/16181459/200357248-5b17c4e1-8887-47f0-ae4e-b9d97e58b9b7.png)

Still, good catch!

Reply:
Ah! This is pretty common (although not universal) with "short" options (meaning, a single dash followed by a single letter, followed directly by the argument). For example, `git -amwip` is really parsed as `git -a -m "wip"`. I didn't know argparse supported this pattern.

The other test was more broken, though: when I tested it in the terminal, I got different error messages with spaced and non-spaced arguments (but still the same exit code, which is what the test looks at).